.github/workflows/code-check.yml

```diff
@@ -22,23 +22,36 @@ jobs:
         with:
           version: 7
 
-      - name: Get pnpm store directory
+      - name: "Get pnpm store directory"
```

Comment:
🎉 Loving the consistency here. I'm wondering if it's worth adding a Lint rule to automate this? https://www.npmjs.com/package/eslint-plugin-yml (`"double"` is the default)